src/helpers.js

```diff
@@ -442,6 +456,15 @@ export function extractParametersFromNode(schemaNode) {
   const presentation = pickXKey(schemaNode, 'presentation') ?? {};
   const errorMessage = pickXKey(schemaNode, 'errorMessage') ?? {};
   const requiredValidations = schemaNode['x-jsf-logic-validations'];
+  const computedAttributes = schemaNode['x-jsf-logic-computedAttrs'];
+
+  // This is when a forced value is computed.
+  const decoratedComputedAttributes = {
```

Comment:
**Nitpick:** Could this be done in its own function? There's a lot of conditional logic happening in these 4 lines?

Reply:
Updated